docs/index.md

```diff
@@ -92,7 +92,9 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 ## System requirements
 
-Currently, _fastlane_ is officially supported to run on macOS. Support for other operating systems is limited and untested at this point in time. _fastlane_ uses system APIs that may not be implemented on other platforms, for example, we use the Ruby `fork` method for sub-process management, which isn't available on Windows.
+Currently, _fastlane_ is officially supported to run on 🍎 macOS. 
+
+But we are actively working on 🐧 Linux and 🖥️ Windows support for parts of _fastlane_. Some underlying software like Xcode or iTunes Transporter is only available for macOS, but many other tools and actions can theoretically also work on other platforms. Please see [this Github issue for further information](https://github.com/fastlane/fastlane/issues/11687).
```

Comment:
Maybe we should not mention `actively`, as it might sound like that's our #1 priority

Reply:
Good point, I tried to avoid to create any expectations but missed this term. Removed it.